Resources/Translations/fr.lproj/Localizable.strings

```diff
@@ -413,6 +413,7 @@
 "yesterday" = "Hier";
 
 // Alerts
+"please_wait_sync_ending" = "Veuillez Attendre la Fin de la Synchronisation";
```

Comment:
Why capitalize every word? Why not `Veuillez attendre la fin de la synchronisation`?

Reply:
We're following interfaces guidelines, it's common that on iOS important words are capitalized on alerts. You can see an example [here](https://developer.apple.com/design/human-interface-guidelines/ios/views/alerts/)

Reply:
Interfaces guideline state: `Use sentence-style capitalization and appropriate punctuation for complete sentences.`
If this had been "Synchronisation En Cours" I guess, but for a full sentence you need sentence case, not title case.

Reply:
My bad, I should have read the whole page